.github/workflows/build_starter.yml

```diff
@@ -0,0 +1,120 @@
+# Workflow to handle triggering the build process
+name: Firebase Unity SDK build
+
+on:
+  workflow_dispatch:
+    inputs:
+      unity_version:
+        description: 'Unity version'
+        default: '2019'
+        required: true
+      firebase_cpp_sdk_version:
+        description: 'Firebase CPP SDK version to build against (The branch, tag or SHA to checkout)'
+        default: ''
+        required: false
+      unity_branch:
+        description: 'Unity branch to build against, empty means current branch'
+        default: ''
+      platforms:
+        description: 'CSV of Android,iOS,Windows,macOS,Linux'
+        default: 'Android,iOS,Windows,macOS,Linux'
+        required: true
+      apis:
+        description: 'CSV of apis to build and test'
+        default: 'analytics,auth,crashlytics,database,dynamic_links,firestore,functions,installations,messaging,remote_config,storage'
+        required: true
+      # Additional CMake flags to use
+      additional_cmake_flags:
+        description: 'Additional flags to pass into CMake'
+        default: ''
+        required: false
+
+jobs:
+  build_sdks:
+    name: build-${{matrix.platform}}-${{ github.event.inputs.unity_version }}-CPP${{ github.event.inputs.firebase_cpp_sdk_version }}
+    runs-on: ${{matrix.os}}
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [android, ios, macos, windows, linux]
+        include:
+        - platform: android
+          os: macos-latest
+          reuse_action: ./.github/workflows/build_android.yml
+          unity_platform_name: Android
+        - platform: ios
+          os: macos-12
+          reuse_action: ./.github/workflows/build_ios.yml
+          unity_platform_name: iOS
+        - platform: macos
+          os: macos-latest
+          reuse_action: ./.github/workflows/build_macos.yml
+          unity_platform_name: macOS,iOS
+        - platform: windows
+          os: windows-2019
+          reuse_action: ./.github/workflows/build_windows.yml
+          unity_platform_name: Windows
+        - platform: linux
+          os: ubuntu-latest
+          reuse_action: ./.github/workflows/build_linux.yml
+          unity_platform_name: Linux
+    env:
+        # LC_ALL, LANG and U3D_PASSWORD are needed for U3D.
```

Comment:
Are these still needed, since this one doesn't need U3D?

Reply:
removed